ch data acquisition - restructure section: handle data securely

kbjarkefur opened the pull request:
This PR includes a re-write of the _Handling Data Securely_ section. I only had two small tasks, and they both related to how to include non-survey data sources to this section. 

Before this re-write the section completely followed the use case of survey data. I tried and tried to add other sources with only small edits,, but I couldn't, and the re-write in this PR is the result of that.

Apart from breaking up the heavy focus on survey data I also wanted move all parts that explained encrytpion to an encryption sub-section. It was weird that, for example, encryption-at-rest, was explained in the data collection sub-section when that is just as relevant in the data storage or data sharing section.

I feel that this version needs some polishing, but I wont be able to make more progress until I at least get some feedback.

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+The most common reason a data is confidential is that it includes personal identifiers. 
+Removing identifying information is called de-identification,
+and we will discuss how to do that in the next chapter. 
+De-identifying the data, at the earliest possible opportunity 
```
```suggestion
De-identifying the data at the earliest possible opportunity 
```

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+and we will discuss how to do that in the next chapter. 
+De-identifying the data, at the earliest possible opportunity 
+will simplify the workflow without compromising data security,
+as de-identified data no longer must be encrypted.
```
```suggestion
as de-identified data no longer needs to be encrypted.
```

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+De-identifying the data, at the earliest possible opportunity 
+will simplify the workflow without compromising data security,
+as de-identified data no longer must be encrypted.
+Another method of making data less confidential is to aggregate the data, 
```
```suggestion
Another method to protect confidential data is to aggregate the data, 
```

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+and only those numbers are reported.
+Any of these techniques can be used either before the data is shared with the research team,
+or when creating the second non-confidential version of the data set to be used in the analysis.
+If confidential information is directly required for the analysis itself,
```
```suggestion
If confidential information is directly required for the analysis,
```

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+by collecting or receiving data without any confidential information in the first place.
+This should always be your first option when possible.
+
+Usually we need to deal with confidential data at some point in the project, 
```
For the final review, we need to make sure we use the terms 'confidential' and 'personally-identifying" consistently throughout all sections/chapters

kbjarkefur replied:
Ok, opened up an issue for that here: https://github.com/worldbank/d4di/issues/454

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+If confidential information is directly required for the analysis itself,
+it will be necessary to keep at least a subset of the data encrypted through the data analysis process.
+
+There are multiple options on how to share data encrypted. 
```
```suggestion
There are multiple options for sharing encrypted data. 
```

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+
+There are multiple options on how to share data encrypted. 
+If you need to collaborate on a confidential data set 
+then you can create an encrypted folders using VeraCrypt as described above.
```
```suggestion
then you can create an encrypted folder, for example using VeraCrypt (as described above).
```

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+There are multiple options on how to share data encrypted. 
+If you need to collaborate on a confidential data set 
+then you can create an encrypted folders using VeraCrypt as described above.
+An encrypted folder can be shared the same way you share a normal file
+and it can even be shared using insecure modes of communication
+such as email or third-party syncing services.
```
Not clear to me how you would normally share files outside of email / sync software; suggest simplifying the wording as follows: 
An encrypted folder can safely be shared using insecure modes of communication such as email or third-party syncing services.

kbjarkefur replied:
implemented as suggested in e79e29f9ac376bfea501f95976ce45aaafddf1e6

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+
+If you only need a one-time transfer of a confidential dataset, 
+for example when receiving data from a partner organization,
+then the setting up of an encrypted folder
```
```suggestion
then setting up an encrypted folder
```

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+If you only need a one-time transfer of a confidential dataset, 
+for example when receiving data from a partner organization,
+then the setting up of an encrypted folder
+might not be your best option or even feasible.
```
why wouldn't it be feasible?

kbjarkefur replied:
It might be a bit too techy for some counterparts, and I did not know how to express that without either:
1. Make it sound as if encryption and VeraCrypt 
1. Make it sound as if we think that partner organization cannot be techy.

I will go over the other comments and come back to this to think of a better way to say it

kbjarkefur replied:
Made a new try based on a suggestion from Ben in, 9135f9b874c191761db5e2038252ac55504a5395. This says the same thing I want to say, without implying anything about partner organizations.

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+for example when receiving data from a partner organization,
+then the setting up of an encrypted folder
+might not be your best option or even feasible.
+Instead, for one-time transfers you can send data over password protected
```
```suggestion
Instead, for one-time transfers we recommend sending data over password-protected
```

mariaruth commented on chapters/data-collection.tex:
```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
 there are two on-site copies of the data and one off-site copy,
 so the data can never be lost in case of hardware failure.\sidenote{
-  \url{https://www.backblaze.com/blog/the-3-2-1-backup-strategy}}
+	\url{https://www.backblaze.com/blog/the-3-2-1-backup-strategy}}
```
I had removed that url because it is already cited in ch2

kbjarkefur replied:
Might have used outdated text here, as I do did not intentionally add it back. Anyways, removed in 775e293fb88cff27dfbbc11fe97d8184b7ef4f29

kbjarkefur commented:
@mariaruth , I think I have addressed all your comments, so I think this PR can be merged. We only need to hold off with PRs that are merged to the develop branch, so if you are ok with edits, please just merge this.
